script/testing/junit/utils.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/python3
+
+import argparse
+
+from oltpbench import constants
```

Comment:
Circular dependency?

Reply:
Duplicate `import argparse ` in `run_junit.py` deleted